custom_model_runner/datarobot_drum/resource/chat_helpers.py

```diff
@@ -0,0 +1,2 @@
+def is_streaming_response(response):
+    return getattr(response, "object", None) != "chat.completion"
```

Comment:
Make sure to have Copyright block at top of this new file

Reply:
Done